src/RPackage-Tests/RPackageOrganizerSystemTest.class.st

```diff
@@ -0,0 +1,31 @@
+"
+I am a test case to make sure the Smalltalk environment work as intended with its system package organizer.
+"
+Class {
+	#name : #RPackageOrganizerSystemTest,
+	#superclass : #TestCase,
+	#category : #'RPackage-Tests-Image'
+}
+
+{ #category : #tests }
+RPackageOrganizerSystemTest >> testDefault [
+
+	self assert: RPackageOrganizer default identicalTo: self class environment organization
+]
+
+{ #category : #tests }
+RPackageOrganizerSystemTest >> testDefaultEnvironment [
+	"When no environment is specified, point to the default one"
+
+	self assert: RPackageOrganizer new environment identicalTo: Smalltalk globals
+]
+
+{ #category : #tests }
+RPackageOrganizerSystemTest >> testThatExistingPackageNamesDoesNotContainIllegalCharacters [
+	| illegalCharacters |
+
+	illegalCharacters := #( $\ $/ $: $* $? $" $< $> $| ).
```

Comment:
so `;` or `¿` are fair-game? Or starting with ` -` is OK?
I would prefer a whitelist (alphanum and `-`?) and that a test predicate is introduced outside the tests so that tools, rules or whatever can rely on the same policy.
Since there is already `String>>#isValidGlobalName` and `String>>#isValidSelector`, having `String>>#isValidPackageName` makes sense

Reply:
I'm not a big fan of this but here I'm just migrating an existing test :)
We can iterate on this in future PRs

Reply:
Oh, right. It was in the deleted file. I'm fine with future PR then